### DISCUSSION TITLE
how do we change an environment variable?

### DISCUSSION THREAD

**Discussion by PegaFox** [ptitSeb/box86#344] [Q&A]
I'm running box86 on a raspberry pi and when I try to change a variable I get this message:
Box86 with Dynarec v0.2.1 0eaa1b6c built on Jan 25 2021 00:09:45
Error: file is not found (check BOX86_PATH)

**Comment by ptitSeb** [comment]
if you installed box86 (with `sudo make install`), then you can "forget" box86, it's in the system, so just run you program as if it was an arm one.
What are you trying to run?
